crates/ruff_annotate_snippets/src/renderer/display_list.rs

```diff
@@ -253,6 +253,11 @@ impl DisplaySet<'_> {
             if let Some(id) = annotation.id {
                 if hide_severity {
                     buffer.append(line_offset, &format!("{id} "), *stylesheet.error());
+                    if annotation.is_fixable {
+                        buffer.append(line_offset, "[", stylesheet.none);
+                        buffer.append(line_offset, "*", stylesheet.help);
+                        buffer.append(line_offset, "] ", stylesheet.none);
+                    }
```

Comment:
Do we need to gate this behind hide_severity or could we always render it after the severity/and id (unconditionally) if `is_fixable` is true